packages/react-components/src/templates/ProfileHeader.tsx

```diff
@@ -67,16 +67,17 @@ const iconStyles = css({
 
 type ProfileProps = Pick<
   UserResponse,
+  | 'avatarURL'
   | 'department'
   | 'displayName'
-  | 'institution'
-  | 'orcidLastModifiedDate'
   | 'firstName'
+  | 'institution'
+  | 'jobTitle'
+  | 'lastModifiedDate'
   | 'lastName'
   | 'location'
+  | 'orcidLastModifiedDate'
```

Comment:
probably unused now? But I guess the red CI likely already complained :sweat_smile: